controllers/component_build_controller_pac.go

```diff
@@ -373,10 +373,10 @@ func (r *ComponentBuildReconciler) UndoPaCProvisionForComponent(ctx context.Cont
 }
 
 // getPaCWebhookTargetUrl returns URL to which events from git repository should be sent.
-func (r *ComponentBuildReconciler) getPaCWebhookTargetUrl(ctx context.Context, repositoryURL string) (string, error) {
+func (r *ComponentBuildReconciler) getPaCWebhookTargetUrl(ctx context.Context, repositoryURL string, checkConfig bool) (string, error) {
 	webhookTargetUrl := os.Getenv(pipelinesAsCodeRouteEnvVar)
 
-	if webhookTargetUrl == "" {
+	if webhookTargetUrl == "" && checkConfig {
```

Comment:
We can more meaningful name for `checkConfig`

Reply:
changed